_topics/en/silent-payments.md

```diff
@@ -73,5 +73,30 @@ excerpt: >
 
 ---
 
+Traditionally, a new address should be generated whenever a user receives
+two or more payments. This is because when you receive several payments
+to the same Bitcoin address, others can reasonably assume that the same
+person received all of those payments even if the payments are later
+spent in separate transactions. This is also known as address reuse,
```

Comment:
The big problem with address reuse is that it not only hints at someone being in control of multiple UTXOs such as with the [Common-input-ownership heuristic](https://en.bitcoin.it/wiki/Common-input-ownership_heuristic), but since it’s actually the same output script, it’s undeniable that the funds went to the same receiver.

```suggestion
to the same Bitcoin address, it is a given that the same
person or entity received all of those payments even if the outputs are later
spent in separate transactions. This is also known as address reuse,
```